bindings/rust/build.rs

```diff
@@ -2,14 +2,39 @@ fn main() {
     let src_dir = std::path::Path::new("src");
 
     let mut c_config = cc::Build::new();
-    c_config.include(src_dir);
+    c_config.include(&src_dir);
     c_config
         .flag_if_supported("-Wno-unused-parameter")
         .flag_if_supported("-Wno-unused-but-set-variable")
         .flag_if_supported("-Wno-trigraphs");
     let parser_path = src_dir.join("parser.c");
     c_config.file(&parser_path);
 
+    // If your language uses an external scanner written in C,
```

Comment:
Hey @amaanq, would you mind reviewing this draft before I turn it into a PR? I ran `npm run build` to regenerate the bindings but I see it included a bunch of commented blocks, including some that look like they should be preserved. See https://github.com/tree-sitter-grammars/tree-sitter-capnp/pull/19/files#diff-f1aca5f9358053e570c736ccd84ad4e75fc98048bb3bfbb16fa173d4555f48c5L31-L49